File: java-back-server/src/main/java/space/borisgk/taxi/api/model/entity/AuthServiceData.java
package space.borisgk.taxi.api.model.entity;

import lombok.*;
import org.hibernate.validator.constraints.Range;
import space.borisgk.taxi.api.model.AuthService;
import space.borisgk.taxi.api.model.IEntity;

import javax.persistence.*;
import javax.validation.Constraint;

@Entity
@Setter
@Getter
@Builder
@AllArgsConstructor
@NoArgsConstructor
@Table(name = "auth_service_data")
public class AuthServiceData implements IEntity<Integer> {

    @Id
    @GeneratedValue(strategy = GenerationType.IDENTITY)
    private Integer id;

    private AuthService authService;
    @Column(name = "social_id")
    private String socialId;

    @Column(
            columnDefinition = "BIGINT COMMENT " +
            "'Hash for all user in this social network for user. " +
            "Compute as the sum modulo 2^64 of users\' ids." +
            "This field should be interpreted as a UNSIGNED LONG.'",
            name = "friends_hash"
    )
    @Range(min = 0)
    private Long friendsHash;
}

File: kotlin-back-server/src/main/kotlin/space/borisgk/taxi/api/model/entity/AuthServiceData.kt
package space.borisgk.taxi.api.model.entity

import javax.persistence.Entity
import javax.persistence.GeneratedValue
import javax.persistence.GenerationType
import javax.persistence.Id

@Entity
class AuthServiceData {

    constructor(id: Int?) {
        this.id = id
    }

    @Id
    @GeneratedValue(strategy = GenerationType.IDENTITY)
    var id: Int? = null
        set(id) {
            field = this.id
        }

    var service: AuthService? = null
        set(service) {
            field = this.service
        }

    var serviceToken: String? = null
        set(serviceToken) {
            field = this.serviceToken
        }
}